python/assistant-stream/src/assistant_stream/serialization/data_stream.py

```
@@ -22,7 +22,8 @@ def encode_chunk(self, chunk: AssistantStreamChunk) -> str:
             return f'a:{json.dumps({ "toolCallId": chunk.tool_call_id, "result": chunk.result })}\n'
         elif chunk.type == "data":
             return f"2:{json.dumps([chunk.data])}\n"
-        pass
+        elif chunk.type == "error":
+            return f"3:{json.dumps(chunk.error)}\n"
```

Ruff (0.8.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)